src/shared/LayoutContainer.tsx

```diff
@@ -44,7 +45,7 @@ export const LayoutContainer = ({
             EDUGATOR
           </Typography>
           <Box sx={{ flexGrow: 1 }} />
-          {history.location.pathname !== Routes.Login.toString() && (
+          {LocalStorage.getToken() && (
```

Comment:
This reintroduces the bug that when the session token expires, the logout button disappears, and the user is forced to refresh the page.

See one of your previous pull requests where you changed this line of code.